README.md

```diff
@@ -42,3 +43,28 @@ express.
 
 On a successful LTI launch the server returns a signed `accessToken` jwt that
 the client can then later use to authenticate saving content.
+
+# Management of Env Vars of the Deployments
```

Comment:
Should this be done on the server or locally? Maybe add a comment about this here.

Reply:
I've updated the readme. Take a look if it is clearer now please.

Reply:
Now it is clear! Thank you for updating it! :green_heart: